core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -61,23 +63,25 @@ def prepare_options
       @run_migrations = options[:migrate]
       @load_seed_data = options[:seed]
       @load_sample_data = options[:sample]
+      @selected_frontend = detect_frontend_to_install
+      @selected_authentication = detect_authentication_to_install
 
       # Silence verbose output (e.g. Rails migrations will rely on this environment variable)
       ENV['VERBOSE'] = 'false'
 
       # No reason to check for their presence if we're about to install them
       ENV['SOLIDUS_SKIP_MIGRATIONS_CHECK'] = 'true'
 
-      if options[:with_authentication] != nil
+      if options[:enforce_available_locales] != nil
         warn \
-          "DEPRECATION WARNING: using `solidus:install --with-authentication` is now deprecated. " \
-          "Please use --authentication instead (see --help for the full list of options)."
+          "DEPRECATION WARNING: using `solidus:install --enforce-available-locales` is now deprecated and has no effect. " \
```

Comment:
Why not remove the option without deprecating it first if it has no effect? The generator will just ignore unknown options.

Reply:
`Rails::Generators::AppBase` is capable of accepting one argument as the app name/place (like in `rails new`), we don't need to specify it (defaults to `Rails.root`) but that also means that any unrecognized option will be interpreted as it. 

I would also argue that any existing script with formerly supported arguments (e.g. sandbox scripts) will be better off being notified it's no longer a supported option.

Reply:
Yeah, that's a good point :+1: